src/NServiceBus.Core/Routing/MessageDrivenSubscriptions/PublisherAddress.cs

```diff
@@ -58,7 +59,8 @@ internal IEnumerable<string> Resolve(Func<Endpoint, IEnumerable<EndpointInstance
             {
                 return instances.Select(addressResolver);
             }
-            return instanceResolver(endpoint).Select(addressResolver);
+            var result = await instanceResolver(endpoint);
```

Comment:
`ConfigureAwait(false)`
Is there no way we can turn this into a return without await?